src/SFA.DAS.PR.Data.UnitTests/Repositories/NotificationsWriteRepositoryTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+using SFA.DAS.PR.Data.Repositories;
+using SFA.DAS.PR.Data.UnitTests.InMemoryDatabases;
+using SFA.DAS.PR.Data.UnitTests.Setup;
+using SFA.DAS.PR.Domain.Common;
+using SFA.DAS.PR.Domain.Entities;
+
+namespace SFA.DAS.PR.Data.UnitTests.Repositories;
+
+public class NotificationsWriteRepositoryTests
+{
+    private readonly CancellationToken cancellationToken = CancellationToken.None;
+
+    [Test]
+    public async Task CreateNotifications_Returns_Success()
+    {
+        Notification[] notifications = [NotificationsTestData.Create(Guid.NewGuid(), NotificationType.Employer)];
```

Comment:
Since this is saving multiple records, I would actually add one more here with a different type. You can use fixture to create this without any helpers. There is a `WithValues` and `CreateMany` extension that I have added into testing library that will come handy here.